notebooks/timeseries-conversion.ipynb

```diff
@@ -106,13 +276,236 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "target_times = np.array([-5, 5, 10, 20, 30, 40, 45, 50, 60])\n",
+    "examples = [\n",
+    "    (\n",
+    "        \"Linear extrapolation\", \n",
+    "        ExtrapolationType.LINEAR, \n",
+    "        {\"fill_value\": \"extrapolate\"}\n",
+    "    ),\n",
+    "    (\n",
+    "        \"Constant extrapolation\", \n",
+    "        ExtrapolationType.CONSTANT, \n",
+    "        {\"fill_value\": (source_values[0], source_values[-1]), \"bounds_error\": False}\n",
+    "    ),\n",
+    "]\n",
+    "for (title, extrap_type, numpy_kwargs) in examples:\n",
+    "    point_target_view = paras.get_timeseries_view(\n",
+    "        (\"Emissions\", \"CH4\"),\n",
+    "        (\"World\",),\n",
+    "        \"MtCH4 / yr\",\n",
+    "        target_times,\n",
+    "        timeseries_type=ParameterType.POINT_TIMESERIES,\n",
+    "        interpolation_type=InterpolationType.LINEAR,\n",
+    "        extrapolation_type=extrap_type,\n",
+    "    )\n",
     "\n",
-    "If your data is periodic, all we need to do to specify our timeseries is to provide the start time and period length of the time a
```

Comment:
```suggestion
    "           \"/\", \n",
```

Reply:
???

Reply:
no i get it now my bad

Reply:
Oh, what happened here? Did I review on outdated source?

Reply:
I think so...